deploy/cephfs/kubernetes/csi-config-map.yaml

```diff
@@ -12,4 +12,4 @@ metadata:
   name: "ceph-csi-config"
 data:
   config.json: |-
-    []
+    [{Dmeo [] {   } { } {} {true [a b]}}]
```

Comment:
Dmeo?

This format does not look valid, so the generation isn't setup correctly. I don't know where this comes from, but the values need to be set so that the file can be applied directly.

Reply:
oops!, Just for testing I had modified in config map,
this got added, didn't notice it
Apologies